run_aprime.bash

```diff
@@ -353,35 +343,48 @@ if [ ! -d $www_dir/$plots_dir_name ]; then
 else
   rm -f $www_dir/$plots_dir_name/*
 fi
+# Check on MPAS-Analysis www_dir, permissions included
+# Create it if it does not exist, purge it if it does
+export mpas_www_link=mpas-analysis # mpas_www_link is used by generate_html to define
+                                   # the html link to the MPAS-Analysis web interface
+if [ ! -d $www_dir/$mpas_www_link ]; then
+  mkdir -p $www_dir/$mpas_www_link
+  chmod -R ga+rX $www_dir/$mpas_www_link
+fi
+export mpas_www_link=$mpas_www_link/$plots_dir_name
+export mpas_www_dir=$www_dir/$mpas_www_link
+if [ ! -d $mpas_www_dir ]; then
+  mkdir -p $mpas_www_dir
+else
+  rm -rf $mpas_www_dir/*
+fi
 
 # LOAD THE MACHINE-SPECIFIC ANACONDA-2.7 ENVIRONMENT
 source $MODULESHOME/init/bash  
 if [ $machname == "nersc" ]; then
   module unload python
   module unload python_base
   module use /global/project/projectdirs/acme/software/modulefiles/all
-  module load python/anaconda-2.7-acme
-  export NCO_PATH_OVERRIDE=No
+  module load e3sm-unified/1.1.3
+  #export NCO_PATH_OVERRIDE=No
```

Comment:
commenting this out has worked for me on edison, so perhaps it is no longer necessary.

Reply:
I would not take this out.  If you do, I believe you will still be using @czender's version of NCO which might happen to be compatible with e3sm-unified/1.1.3 right now but might not be in the very near future!

Reply:
ok, I will leave it. Right now it has worked on edison, but better leave it for future versions.